pkg/status/render.go

```diff
@@ -96,6 +96,12 @@ func FormatStatus(data []byte) (string, error) {
 		}
 		return nil
 	}
+	remoteConfigFunc := func() error {
```

Comment:
WDYT about always displaying the remote config section in the status page but with a message when disabled ?

We do this for other feature:
```==========
Logs Agent
==========

  Logs Agent is not running


=============
Process Agent
=============

  Status: Not running or unreachable
```